SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -23,7 +27,7 @@ Your folder structure should now resemble -
 \<ProjectRoot>\LaunchClient.bat 
 etc...
 ```
-1. Open **`ProjectPaths.bat`** for editing and replace `Game` with `<GameRoot>`, and `StarterProject` with the name of your uproject (which we'll refer to as `<YourProject>`).
+1. Open **` \<ProjectRoot>\ProjectPaths.bat`** for editing and replace `Game` (located at `set PROJECT_PATH=Game`) with your `<GameRoot>` folder name, and `StarterProject` (located at `set GAME_NAME=StarterProject`) with the name of your games uproject (which we'll refer to as `<YourProject>`).
 
 ## Adding the SpatialGDK module to your project
 1. In your project's `*.build.cs` file, add "SpatialGDK" to the `PublicDependencyModuleNames`.
```

Comment:
The snippet below seems to be borked. 

https://docs.improbable.io/unreal/pre-alpha/content/porting-unreal-project-to-gdk

Reply:
Added an arbitrary code specifier to try and fix it.